hw/ip/usbdev/dv/usbdev_sim_cfg.hjson

```diff
@@ -74,8 +74,13 @@
       uvm_test_seq: usbdev_enable_vseq
     }
     {
-      name: usbdev_fifo_rst
-      uvm_test_seq: usbdev_fifo_rst_vseq
+      name: usbdev_in_status_stage
+      uvm_test_seq: usbdev_in_status_stage_vseq
+    }
+    {
+      name: usbdev_in_trans
+      uvm_test_seq: usbdev_in_trans_vseq
+>>>>>>> [usbdev] in status stage sequence
```

Comment:
Nit: Something has gone a bit wrong with the merging here. Could you tidy it up?